### DISCUSSION TITLE
Rewrite_URLS='0'

### DISCUSSION THREAD

**Discussion by RAMINCNZ** [miqrogroove/family-tree-home#6] [Q&A] [answered]
What if one wants to keep pretty URLs??

Can the module be updated to allow for that??

Rick M
Nu Tirani - New Zealand

**Comment by miqrogroove** [comment, marked as answer]
Hi Rick, thank you for writing with a question.  This module should work perfectly with pretty URLs.  My own site works that way.  Let me know if you experience any new issues with it.
